input/pages/quality-improvement-ecosystem.md

```diff
@@ -5,7 +5,7 @@ The diagram below shows the quality improvement ecosystem.  Each step is discuss
 <br>
 Step 1: Researcher and public health surveillance – this step begins with information, preferably evidence-based from research, public health surveillance, and data mining and other analyses performed by third parties such as academic institutions or payers. Such information indicates existing status and knowledge about a given clinical topic.<br>
 <br>
-Step 2: Clinical practice guidelines – in this step stakeholders, such as professional societies, public health agencies, and governmental bodies, publish such information to assure awareness among consumers, healthcare practitioners, and healthcare organizations about what is known and suggested methods for managing the clinical topic. Ideally, suggested management efforts are captured and documented in guidelines based on collaboration among clinical subject matter experts, terminologists, informaticists, clinicians and consumers.<br>
+Step 2: Clinical practice guidelines – in this step stakeholders, such as professional societies, public health agencies, and governmental bodies, publish such information to provide current recommendations among consumers, healthcare practitioners, and healthcare organizations about what is known and suggested methods for managing the clinical topic. Ideally, suggested management efforts are captured and documented in guidelines based on collaboration among clinical subject matter experts, terminologists, informaticists, clinicians and consumers.<br>
```

Comment:
The changes applied are incomplete and do not align with the proposed wording in the resolution. 

Note that this should be "provide current recommendations to..." instead of "provide current recommendations among ..."

"...... to provide current recommendations to consumers, healthcare practitioners, and healthcare organizations about what is known and suggested methods for managing the clinical topic."